terraform.tfvars

```diff
@@ -1,7 +1,7 @@
-org_name   = ""
```

Comment:
Need to provide some input variables as mentioned by the lab [here](https://github.com/DevOpsTestLab/DevOpsNinjas#instructions).